src/app/pages/PerpetualPage/utils/pairsUtils.ts

```diff
@@ -0,0 +1,9 @@
+import { PerpetualPairDictionary } from 'utils/dictionaries/perpetual-pair-dictionary';
+import { PerpetualPair } from 'utils/models/perpetual-pair';
+
+export const perpIds = PerpetualPairDictionary.list().map(pair => pair.id);
```

Comment:
This isn't used anywhere so you can delete the whole file. Even if it was used, I wouldn't create an extra file for that alone, I would consider adding it to `PerpetualPairDictionary` itself (but it's not necessary at the moment).